### PR TITLE
Reconcile adam.lua with the latest version of the paper

### PR DESCRIPTION
**Pull request by AjayTalati**
Some cosmetic changes to make the hyper-parameters the same as the latest version of the paper

### CODE REVIEW

File: adam.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
---[[ An implementation of Adam http://arxiv.org/pdf/1412.6980v2.pdf
+--[[ An implementation of Adam http://arxiv.org/pdf/1412.6980.pdf
```
Comment (soumith) [inline adam.lua]: http://arxiv.org/abs/1412.6980 instead, so that people always get the latest paper draft

**Comment by soumith** [comment]
cc: @fidlej who is good at this stuff, can you review this small diff.

**Comment by AjayTalati** [comment]
The current version of `adam.lua` uses default parameters, 

`learningRate = 2e-4 ,beta1 = 0.1, beta2 = 0.001`

The new version of the paper uses, default values of 

 `beta1 = 0.9, beta2 = 0.999`

The changes I made are simply of the form 

`beta_new <- 1- beta_old`

**Comment by fidlej** [comment]
The changes are good. Thanks Ajay for updating adding the test.

**Pull request merged**